Sources/PackageSyntax/Formatting.swift

```diff
@@ -0,0 +1,210 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2021 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import SwiftSyntax
+
+extension ArrayExprSyntax {
+    public func withAdditionalElementExpr(_ expr: ExprSyntax) -> ArrayExprSyntax {
```

Comment:
lets audit what must be public and what can remain internal / private in order to control the public API surface area